src/test/kotlin/uk/gov/dluhc/printapi/rds/entity/CertificateTest.kt

```diff
@@ -12,6 +12,49 @@ import java.time.temporal.ChronoUnit
 
 internal class CertificateTest {
 
+    @Nested
+    inner class GetCurrentPrintRequest {
+
+        @Test
+        fun `should get latest print request for Certificate with no Print Requests`() {
+            // Given
+            val certificate = certificateBuilder(printRequests = listOf())
```

Comment:
perhaps `emptyList()` ? 😉  not a blocker

Reply:
👍  - included in next PR